### DISCUSSION TITLE
How to delete a permanent controller?

### DISCUSSION THREAD

**Discussion by AthreshK** [jonataslaw/getx#2617] [Q&A]
I have a controller that has been put as permanent.
` Get.put(SessionController(),permanent: true);`

When I try to delete it with 
`Get.delete<SessionController>();`

I get the error 
`"SessionController" has been marked as permanent, SmartManagement is not authorized to delete it.`

I want to keep the controller between a few named routes so I don't want it to get automatically deleted unless I do it myself.

**Comment by jonataslaw** [comment]
Get.delete<SessionController>(force:true);
